spark-jobs/src/main/scala/filodb/downsampler/index/DSIndexJobSettings.scala

```diff
@@ -28,6 +28,9 @@ object DSIndexJobSettings extends StrictLogging {
 
   val cassWriteTimeout = dsIndexJobConfig.as[FiniteDuration]("cassandra-write-timeout")
 
+  //default 6hours
+  val batchLookbackInHours = dsIndexJobConfig.as[Option[Int]]("batch-lookback-in-hours").getOrElse(6)
```

Comment:
Dont hard-code 6.
It is flush-interval of downsample store. Should be in settings.

Reply:
ack, fixed it